source/payment_flow/index.html.md.erb

```diff
@@ -102,7 +102,7 @@ For example:
 }
 ```
 
-`amount`: The amount in pence - in the example, the payment is for £145.
+`amount`: The amount in pence - in the example, the payment is for £145. This value must be a number data type.
```

Comment:
I wonder if explicitly putting that it should be a number and not a string might be useful (since that’s the most common way people mess this up)

Reply:
If I change are you happy for it to be merged?

Reply:
Sure